company-keywords.el

```diff
@@ -297,6 +297,26 @@
     (enh-ruby-mode . ruby-mode))
   "Alist mapping major-modes to sorted keywords for `company-keywords'.")
 
+(with-eval-after-load 'make-mode
+  (require 'seq)
+  (mapc
+   (lambda (mode-stmnts)
+     (setf (alist-get (car mode-stmnts) company-keywords-alist)
+           (seq-uniq
+            (append
+             makefile-statements
```

Comment:
`makefile-automake-statements` and others seem to already include `makefile-statements`.

So I think it should be dropped from here and instead replace the `nil` values below.

Reply:
I'll fix it next push.